cibyl/sources/elasticsearch/api.py

```diff
@@ -85,11 +85,13 @@ def __query_get_hits(self: object, query: dict, index: str = '*') -> list:
         :return: List of hits.
         """
         try:
+            self.es_connection = self.es_client.connect()
             response = self.es_connection.search(
                 index=index,
                 body=query,
-                size=1000
+                size=10000,
             )
+            self.es_connection.transport.close()
```

Comment:
As far as I can tell, 'es_connection' only exists in this chunk of code, so you could remove 'self.' and just have it as a local variable. Also, see if it is possible to use the 'with' keyword, not needed, but could simplify things a little.

Reply:
Now it's using the `with` keyword. Thank you Jorge.